clients/src/main/java/org/apache/kafka/clients/consumer/internals/AsyncKafkaConsumer.java

```diff
@@ -663,6 +662,10 @@ public Map<TopicPartition, OffsetAndMetadata> committed(final Set<TopicPartition
                     time.timer(timeout));
                 committedOffsets.forEach(this::updateLastSeenEpochIfNewer);
                 return committedOffsets;
+            } catch (TimeoutException e) {
+                throw new TimeoutException("Timeout of " + timeout.toMillis() + "ms expired before the last " +
+                    "committed offset for partitions " + partitions + " could be determined. Try tuning default.api.timeout.ms " +
```

Comment:
nit: please replace the literal string `default.api.timeout.ms` with the appropriate constant from `ConsumerConfig` 🥺